.github/workflows/deploy-to-azure.yml

```diff
@@ -0,0 +1,59 @@
+name: DEPLOY-TO-AZURE
+
+on:
+  workflow_dispatch:
+    permissions:
+      teams:
+        - 'reactor/team'
+
+permissions: read-all
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      SERVICE_NAME: ${{ secrets.AZURE_SERVICE_NAME }}
+      APP_NAME: ${{ secrets.AZURE_APP_NAME }}
+      JAR_PATH: ${{ github.workspace }}/build/libs/projectreactor-home.jar
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+      - uses: actions/setup-java@9704b39bf258b59bc04b50fa2dd55e9ed76b47a8 # v4
+        with:
+          distribution: 'temurin'
+          java-version: 8
+      - uses: gradle/gradle-build-action@29c0906b64b8fc82467890bfb7a0a7ef34bda89e # v3
+        name: build
+        with:
+          arguments: build -x spotlessCheck
+      - name: Login via Azure CLI
+        uses: azure/login@v1
```

Comment:
I suppose we should use a specific changeset

Reply:
I was actually expecting dependabot to suggest the correct SHA hash ?